docs/html/topics/configuration.md

```diff
@@ -29,11 +29,15 @@ pip has 4 "levels" of configuration files:
 - `base` : per-base environment configuration file, shared across all virtualenvs with the same base. (available since pip 23.0)
 - `site`: per-environment configuration file; i.e. per-virtualenv.
 
+Additionally, environment variables can be specified which will override any of the above.
```

Comment:
Likely also need to mention command line arguments also override configs.

Reply:
I'm OK to move that section around, if people think that's too late in the file.